.test-infra/jenkins/job_Release_Gradle_NightlySnapshot.groovy

```diff
@@ -36,13 +36,29 @@ job('beam_Release_Gradle_NightlySnapshot') {
       'dev@beam.apache.org')
 
 
-  // Allows triggering this build against pull requests.
+  // Allows triggering this publish command against pull requests.
   common_job_properties.enablePhraseTriggeringFromPullRequest(
       delegate,
       './gradlew publish',
       'Run Gradle Publish')
 
+  // Allows triggering this build command against pull requests.
```

Comment:
This "Run Gradle Build" trigger seems misleading, since it will also publish. Maybe delete it, or set up a separate jenkins job to just build with that trigger phrase.

Reply:
I think 'Run Gradle Build' just triggers './gradlew clean build', which excludes 'publish'

Reply:
Sorry for the misunderstanding. Yes this phrase triggers 'publish'. I think it's better to delete these phrases.